packages/material-ui/src/utils/ownerDocument.ts

```diff
@@ -0,0 +1,3 @@
+export default function ownerDocument(node: Node | undefined): Document {
+  return (node && node.ownerDocument) || document;
```

Comment:
I think this is technically wrong. `ownerDocument` can be `null` if `node` itself is a `Document` in which case we return the `document` in which this module was initialized. These are not necessarily the same, no?

```suggestion
  return node === undefined ? document : (node.ownerDocument ?? node);
```

Though honestly I'd rather not allow passing `undefined` here.

Reply:
Leaving this for another PR.